### PR TITLE
Calculate the year to match the current year (or next year for New Year)

### PR DESCRIPTION
**Pull request by SplinterHead**
This avoids having to keep the script up to date each year

### CODE REVIEW

File: src/the_final_countdown/main.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import datetime
 
+current_year = int(datetime.datetime.now().strftime("%Y"))
```
Comment (pferretti) [inline src/the_final_countdown/main.py]:
```suggestion
current_year = datetime.datetime.now().year
```

File: src/the_final_countdown/main.py [hunk 2]
```diff
@@ -13,9 +14,9 @@ def main():
     print("It's the final countdown")
     print("=" * 80)
     print("Select the date:")
-    print("1) New year's day 2020")
-    print("2) Halloween 2019")
-    print("3) Christmas 2019")
+    print("1) New year's day %d" % (current_year + 1))
```
Comment (pferretti) [inline src/the_final_countdown/main.py]:
```suggestion
    print(f"1) New year's day {current_year + 1}")
```

File: src/the_final_countdown/main.py [hunk 3]
```diff
@@ -13,9 +14,9 @@ def main():
...
+    print("2) Halloween %d" % current_year)
```
Comment (pferretti) [inline src/the_final_countdown/main.py]:
```suggestion
    print(f"2) Halloween {current_year}")
```

File: src/the_final_countdown/main.py [hunk 4]
```diff
@@ -13,9 +14,9 @@ def main():
...
+    print("3) Christmas %d" % current_year)
```
Comment (pferretti) [inline src/the_final_countdown/main.py]:
```suggestion
    print(f"3) Christmas {current_year}")
```

**Review by pferretti — COMMENTED** [review]
Thanks for your PR!
Here are some suggestions :-)

**Comment by SplinterHead** [comment]
Thanks for the suggestions, committed those now

**Pull request closed without merging**